packages/plugins/src/components/plugin-area/index.js

```diff
@@ -58,16 +59,22 @@ class PluginArea extends Component {
 	}
 
 	getCurrentPluginsState() {
-		return {
-			plugins: map( getPlugins(), ( { icon, name, render } ) => {
+		const plugins = compose(
```

Comment:
It would be great to have a test which validates whether components are rendered in the order enforced by priorities.

Reply:
Sounds great! I'll get those in place ASAP.